README.md

```diff
@@ -68,4 +68,17 @@ Guy Itzhaki, Tarel Madar, Gal Lapid, Alon Weissfeld, Omri Rosner
 1. Install VirtualBox and Vagrant.
 2. Clone this repository.
 3. Open a shell prompt, and switch to the location of the repository.
-4. Run 'vagrant up'. Once it finished, you're all set!
```

Comment:
The following does not seem related to login/registration - does this belong in a different PR?

Reply:
Yes, you're right.
I felt it was redundant to open a new PR for such a small change, but I will do so if necessary.